systemdev/kernel-pinephone.rst

```diff
@@ -0,0 +1,71 @@
+PinePhone Kernel
+================
+
+This document describes how to build and install the PinePhone kernel. Before getting to the actual compilation, we describe the PinePhone recovery. At the bottom there are some references relevant for kernel development.
+
+Recovery
+--------
+
+The UBports image contains a recovery boot mode based on `jumpdrive <https://github.com/dreemurrs-embedded/Jumpdrive>`_. To get into the recovery press and hold the power and the volume up buttons until the led turns on.
```

Comment:
Might be relevant to link the recovery source itself: https://github.com/ubports/jumpdrive